app/react/src/client/preview/index.ts

```diff
@@ -10,12 +10,11 @@ const framework = 'react';
 
 interface ClientApi extends ClientStoryApi<StoryFnReactReturnType> {
   setAddon(addon: any): void;
-  configure(loaders: () => void, module: NodeModule): void;
+  configure(loader: Loadable, module: NodeModule, framework: string): void;
```

Comment:
This type seems wrong to me, this is the correct type for `api.configure` but the export of `configure` is already passing the `framework` to `api.configure`.

https://github.com/storybookjs/storybook/blob/86370b51296f62b318404e3f8858ed1762992d24/app/react/src/client/preview/index.ts#L28

If this is not a mistake why would I need to pass the framework when importing from `react/*`?

```ts
import { configure } from '@storybook/react';

function loadStories() {
  // require(stories)
}

configure(loadStories, module, 'react');
```

Thanks 👍 @ shilman

Reply:
@nickofthyme You're right. The exported type is wrong & I'll fix it.

https://github.com/storybookjs/storybook/issues/7595

Reply:
@shilman No worries thanks for the quick response! 👍

Reply:
https://github.com/storybookjs/storybook/pull/7598

Mind reviewing?